backend/src/main/java/codezap/tag/repository/TemplateTagJpaRepository.java

```diff
@@ -22,31 +22,31 @@ public interface TemplateTagJpaRepository extends TemplateTagRepository, JpaRepo
     List<Tag> findAllTagsByTemplate(Template template);
 
     @Query("""
-        SELECT tt, t
-        FROM TemplateTag tt
-        JOIN FETCH tt.tag t
-        WHERE tt.id.templateId = :templateId
-        """)
+            SELECT tt, t
+            FROM TemplateTag tt
+            JOIN FETCH tt.tag t
+            WHERE tt.id.templateId = :templateId
+            """)
     List<TemplateTag> findAllByTemplateId(Long templateId);
 
     @Query("""
-        SELECT tt, t
-        FROM TemplateTag tt
-        JOIN FETCH tt.tag t
-        WHERE tt.id.templateId in :templateIds
-        """)
+            SELECT tt, t
+            FROM TemplateTag tt
+            JOIN FETCH tt.tag t
+            WHERE tt.id.templateId in :templateIds
+            """)
     List<TemplateTag> findAllByTemplateIdsIn(List<Long> templateIds);
 
     @Query("""
-            SELECT DISTINCT t
-            FROM Tag t
-            WHERE t.id IN (
-                SELECT DISTINCT tt.id.tagId
-                FROM TemplateTag tt
-                WHERE tt.id.templateId IN
-                    (SELECT te.id FROM Template te WHERE te.member.id = :memberId)
-        )
-        """)
+                SELECT DISTINCT t
+                FROM Tag t
+                WHERE t.id IN (
+                    SELECT DISTINCT tt.id.tagId
+                    FROM TemplateTag tt
+                    WHERE tt.id.templateId IN
+                        (SELECT te.id FROM Template te WHERE te.member.id = :memberId)
+            )
+            """)
```

Comment:
이 부분은 요렇게 하는게 어떨까요? 그래야 25~37 라인과 일관성이 있을 것 같아용
```suggestion
            SELECT DISTINCT t
            FROM Tag t
            WHERE t.id IN (
                SELECT DISTINCT tt.id.tagId
                FROM TemplateTag tt
                WHERE tt.id.templateId IN (
                   SELECT te.id FROM Template te WHERE te.member.id = :memberId
                )
            )
            """)
```

Reply:
바로 반영 ~!!!